.github/workflows/build_all.yml

```diff
@@ -373,104 +373,196 @@ jobs:
           SLACK_ICON_EMOJI: ":heavy_check_mark:"
           SLACK_COLOR: "#00FF00"
           SLACK_FOOTER: ' '
-  agw-container-build:
+
+  agw-container-build-c-ghz:
+    runs-on: ubuntu-latest
     if: github.repository_owner == 'magma'
-    name: agw container build
+    # The dependency on the regular build is intentional.
+    # There is a hidden dependency in the dockerfile.
+    # As GHZ is not a production build, a more sophisitacted dependency chain
+    # is not worth the effort.
+    needs: agw-container-build-c
+    env:
+      DOCKER_REGISTRY: agw-ci.artifactory.magmacore.org
+      DOCKER_IMAGE: ghz_gateway_c
+      DOCKERCONTEXT: lte/gateway/docker/ghz
+    outputs:
+      digest: ${{ steps.agw-build-docker.outputs.digest }}
+    steps:
+      - run: echo "DOCKER_REGISTRY=agw-test.artifactory.magmacore.org" >> $GITHUB_ENV
```

Comment:
Same as for the `action.yml`: I would prefer if we could use `name:` everywhere, else the "step view" of each job is ugly and hard to understand when looking at the website.

Reply:
No! I am hard against that. Adding a name, does not add any additional value. In fact, it is only reducing the amount of information, you get out of the logs without clicking and expanding the single steps. For single `run` steps, it is much more informative and developer friendly to see the exact command issued. So it is exactly the opposite of what you describe. Additionally this added lines are bloating the code unnecessarily, as they are pure duplication.

Reply:
I disagree, but we can leave it as it is.